cmd/spiffe-aws-assume-role/cli/cli_test.go

```diff
@@ -17,39 +20,62 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestParsesSessionDuration(t *testing.T) {
+var requiredArgs = []string{
+	"credentials",
+	"--audience=foo",
+	"--role-arn=bar",
+	"--spiffe-id=baz",
+}
```

Comment:
In an effort to reduce boilerplate (many of the tests just take the standard required args and add one more parameter) I've extracted the required arguments to their own variable so that they don't have to be repeated over and over.